sdk/core/core-arm/package.json

```diff
@@ -105,8 +105,9 @@
     "integration-test:browser": "echo skipped",
     "integration-test:node": "echo skipped",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "lint:fix": "tslint -p . --fix -c tslint.json --exclude test/**/*.ts",
-    "lint": "tslint -p . -c tslint.json --exclude test/**/*.ts",
+    "lint:fix": "eslint -c ../../.eslintrc.json src tests samples --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint -c ../../.eslintrc.json src tests samples --ext .ts -f node_modules/eslint-detailed-reporter/lib/detailed.js -o core-arm-lintReport.html || exit 0",
```

Comment:
Is the `|| exit 0` at the end what causes lint errors to **not** fail the build?  If so, is the plan to remove this from each package as we verify it is lint clean?

Reply:
Yes the `exit 0` makes it succeed even with lint errors. Yes we can remove this after we verify each package is lint clean.